### ISSUE TITLE
Adding parameters for germline and somatic genomicSourceClass

### ISSUE THREAD

**Issue by fatemaker254** [FHIR/genomics-operations#48]
## Issue
Adding parameters for germline and somatic genomicSourceClass

## Context
Under Subject Genotype Operations the Patient NB6TK328's genotype under finding_subject_haplotype showing same results for both somatic and germline genomicSourceClass parameter.

## Solution
Different output should be displayed after selection of either genomic or somatic genomicSourceClass.

**Comment by fatemaker254** [comment]
Hi  @Rohan-cod  please assign me to this issue.

**Comment by rhdolin** [comment]
Let's defer this one for now - currently, none of the haplotype data in the reference implementation has genomicSourceClass populated, so tightening this logic might be confusing (e.g. searching specifically for 'germline' will return zero results).

**Comment by fatemaker254** [comment]
Yeah got your point @rhdolin . While resolving this doubt I also saw that the actual.json was not getting created properly for specific genomicSourceClass since the searching was not giving any specific output based on that class, I think that was because of this reason only.
